tests/unit/options/unit_tests_sender_option_parser.rb

```diff
@@ -511,6 +511,24 @@ def test_sender_option_parser_user_msg_subject_value_long
     )
   end # test_sender_option_parser_user_msg_subject_value_long
 
+  def test_sender_option_parser_default_anonymous_value
+    sender_options_default_msg_subject = Options::SenderOptionParser.new([])
```

Comment:
Rename variable from *_msg_subject to *_anonymous

Reply:
done